include/clang/Interpreter/CppInterOp.h

```diff
@@ -37,42 +37,57 @@ namespace Cpp {
   using TInterp_t = void*;
   using TCppObject_t = void*;
 
-  enum BinaryOperator {
-    PtrMemD = 0,
-    PtrMemI,
-    Mul,
-    Div,
-    Rem,
-    Add,
-    Sub,
-    Shl,
-    Shr,
-    Cmp,
-    LT,
-    GT,
-    LE,
-    GE,
-    EQ,
-    NE,
-    And,
-    Xor,
-    Or,
-    LAnd,
-    LOr,
-    Assign,
-    MulAssign,
-    DivAssign,
-    RemAssign,
-    AddAssign,
-    SubAssign,
-    ShlAssign,
-    ShrAssign,
-    AndAssign,
-    XorAssign,
-    OrAssign,
-    Comma,
+  enum Operator {
+    OP_None,
+    OP_New,
+    OP_Delete,
+    OP_Array_New,
+    OP_Array_Delete,
+    OP_Plus,
+    OP_Minus,
+    OP_Star,
+    OP_Slash,
+    OP_Percent,
+    OP_Caret,
+    OP_Amp,
+    OP_Pipe,
+    OP_Tilde,
+    OP_Exclaim,
+    OP_Equal,
+    OP_Less,
+    OP_Greater,
+    OP_PlusEqual,
+    OP_MinusEqual,
+    OP_StarEqual,
+    OP_SlashEqual,
+    OP_PercentEqual,
+    OP_CaretEqual,
+    OP_AmpEqual,
+    OP_PipeEqual,
+    OP_LessLess,
+    OP_GreaterGreater,
+    OP_LessLessEqual,
+    OP_GreaterGreaterEqual,
+    OP_EqualEqual,
+    OP_ExclaimEqual,
+    OP_LessEqual,
+    OP_GreaterEqual,
+    OP_Spaceship,
+    OP_AmpAmp,
+    OP_PipePipe,
+    OP_PlusPlus,
+    OP_MinusMinus,
+    OP_Comma,
+    OP_ArrowStar,
+    OP_Arrow,
+    OP_Call,
+    OP_Subscript,
+    OP_Conditional,
+    OP_Coawait,
   };
 
+  enum OperatorArity { kUnary = 1, kBinary, kBoth };
```

Comment:
```suggestion
  enum OperatorArity { kBoth = 0, kUnary = 0, kBinary };
```

Reply:
I did not understand this.
All Unary operators are not Binary operators. So `kBoth = 0` and `kUnary = 0` is confusing. Example `~` operator.

Reply:
Sorry, my bad, fixed the suggestion.